auth/src/main/java/com/firebase/ui/auth/util/data/AuthViewModelBase.java

```diff
@@ -30,4 +36,19 @@ protected FirebaseAuth getAuth() {
     protected PhoneAuthProvider getPhoneAuth() {
         return mPhoneAuth;
     }
+
+    protected CredentialsClient getCredentialsClient() {
+        return mCredentialsClient;
+    }
+
+    @VisibleForTesting
+    public void initializeForTesting(FlowParameters parameters,
```

Comment:
Would like to not need this but I can't see a way to test the ViewModels without it given how ViewModelBase handles initialization.